next/src/components/common/NavBar/NavMenu/NavMenuItem.tsx

```diff
@@ -11,14 +11,24 @@ type NavItemProps = {
   menu: MenuItem
 }
 
+/**
+ * Figma: https://www.figma.com/design/17wbd0MDQcMW9NbXl6UPs8/DS--Component-library?node-id=16846-33525&m=dev
+ */
+
 const NavMenuItem = ({ menu }: NavItemProps) => {
   const category = transformIconToCategory(menu.icon)
   const colorStyle = getCategoryColorLocalStyle({ category })
 
   return (
-    <NavigationMenu.Item>
+    <NavigationMenu.Item className="wrapper-focus-ring rounded-xs">
```

Comment:
Hmm this is interesting that the focus is on `NavigationMenu.Item`, not on `NavMenuTrigger`